wizard/views/load.py

```diff
@@ -0,0 +1,22 @@
+import streamlit as st
+import mlcroissant as mlc
+import tempfile
+from pathlib import Path
+import os
+
+def render_load():
+    col1, col2 = st.columns([1,2], gap="small")
+    with col1:
+        file = st.file_uploader("Select a croissant file to load")
+    if file is not None:
+        try:
+            file_cont = file.read()
+            newfile_name = Path.home() / ".cache" / "croissant" / "loaded_croissant"
```

Comment:
1. `~/.cache/croissant` is already used by [`mlcroissant`](https://github.com/mlcommons/croissant/blob/8d26bdb26b50b3024ed8354eec585822a901bb81/python/mlcroissant/mlcroissant/_src/core/constants.py#L98). (Use `~/.cache/croissant/editor`?)
2. We may want `loaded_croissant` to have a JSON extension.

We can agree on the strategy to manage the cache later.